newrelic_rpm.gemspec

```diff
@@ -47,7 +47,7 @@ https://github.com/newrelic/newrelic-ruby-agent/
   s.homepage = "https://github.com/newrelic/rpm"
   s.require_paths = ["lib"]
   s.summary = "New Relic Ruby Agent"
-  s.add_development_dependency 'feedjira', '3.2.1' unless ENV['CI'] # for Gabby
+  s.add_development_dependency 'feedjira', '3.2.1' unless ENV['CI'] || RUBY_VERSION < '2.5' # for Gabby
```

Comment:
I was wondering if we'd ever have cause to use Ruby 2.4 locally and need to do something like this. Thanks for this fix!